ethereum/tokemak/view_tokemak_PCAs_daily.sql

```diff
@@ -0,0 +1,111 @@
+CREATE MATERIALIZED VIEW tokemak.view_tokemak_PCAs_daily
+(
+    "date", asset_symbol, token_address, total_asset_qty, total_liability_qty, total_liability_value_usd,total_liability_value_eth,total_asset_value_usd,total_asset_value_eth, pca_value_usd, pca_value_eth, pca_qty
+)
+AS (
+    WITH liabilities_daily as (
+        SELECT "date",symbol, pricing_contract, SUM(total_liability_qty) as total_liability_qty
+        FROM tokemak."view_tokemak_outstanding_liabilities_daily" 
+        GROUP BY 1,2,3
+    ),
+    assets_daily as(
+        SELECT "date",symbol, token_address,  SUM(total_qty) as total_qty FROM (
+            SELECT "date", tl.symbol, a.token_address,tl.pricing_contract, total_qty
+            ,tl.is_dollar_stable 
+            FROM tokemak."view_tokemak_all_deployable_assets_by_asset_daily" a 
+            INNER JOIN tokemak."view_tokemak_lookup_tokens" tl on tl.address = a.token_address and is_liability=false 
+        UNION
+        SELECT "date",tl.symbol, token_address, tl.pricing_contract, tokemak_qty as total_qty
+        , tl.is_dollar_stable  
+            FROM (
+            SELECT  b."date", b.wallet_address, b.source_name, b.symbol, tl.address, b.display_name, b.tokemak_qty 
+                FROM tokemak."view_tokemak_wallet_balances_daily" b INNER JOIN 
+                tokemak.view_tokemak_lookup_tokens tl on tl.address = b.token_address AND tl.is_pool = false AND tl.symbol <>'TOKE'
+                LEFT JOIN tokemak."view_tokemak_prices_usd_eth_daily" tp on tp."contract_address" = b."token_address" and tp."date" = b."date"
+                WHERE tl.is_pool = false and tl.symbol <>'TOKE' 
+                ORDER BY source_name, symbol, wallet_address
+            ) b
+            INNER JOIN tokemak."view_tokemak_lookup_tokens" tl on tl.address = b.token_address and is_pool = false and is_liability=false ) as t GROUP BY 1,2,3
```

Comment:
b.token_address doesn't exist in the select within subquery, therefore the join is failing to recognize the column